cms/articles/models.py

```diff
@@ -474,3 +483,39 @@ def get_context(self, request: "HttpRequest", *args: Any, **kwargs: Any) -> dict
         context["has_notices"] = bool(notices)
 
         return context
+
+    def ld_entity(self) -> dict[str, object]:
+        """Add statistical article specific schema properties to JSON LD."""
+        # TODO pass through request to this, once wagtailschemaorg supports it
+        properties = {
+            "url": self.get_full_url(),
+            "headline": self.seo_title or self.listing_title or self.title,
+            "description": self.search_description or self.listing_summary or strip_tags(self.summary),
+            "datePublished": self.release_date.isoformat(),
+            "license": "https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+            "author": {
+                "@type": "Person" if self.contact_details else "Organization",
+                "name": self.contact_details.name if self.contact_details else settings.ONS_ORGANISATION_NAME,
+            },
+            "publisher": {
+                "@type": "Organization",
+                "name": settings.ONS_ORGANISATION_NAME,
+                "url": settings.ONS_WEBSITE_BASE_URL,
+            },
+            "mainEntityOfPage": {
+                "@type": "WebPage",
+                "@id": self.get_full_url(),
+            },
+        }
+        return cast(dict[str, object], extend(super().ld_entity(), properties))
+
+    def get_canonical_full_url(self, request: "HttpRequest") -> str:
+        """Get the article page canonical URL for the given request.
+        If the article is the latest in the series, this will be the evergreen series URL.
+        Otherwise, it will be the default canonical page URL.
+        """
+        canonical_page = self.alias_of.specific if self.alias_of else self
```

Comment:
```suggestion
        canonical_page = self.alias_of.specific_deferred if self.alias_of_id else self
```

as we don't need the full specific page, and by using the `_id` suffix for the foreign key, we avoid an extraneous query if the relation is not set